components/content/src/page.rs

```diff
@@ -567,19 +567,19 @@ And here's another. [^3]
             page.summary,
             Some("<p>This page use <sup>1.5</sup> and has footnotes, here\'s one. </p>\n<p>Here's another. </p>".to_string())
         );
+        // After fixing the duplication bug, page.content should start with continue-reading
+        // and NOT include the summary content again
```

Comment:
That's not correct. `page.summary` has the beginning of the content to display on list pages but you want it part of the content because on the page itself you are not doing `{{page.summary}}{{page.content}}`, you will just do `{{page.content}}` and you expect the full text to be there